src/resmom/mom_comm.c

```diff
@@ -4828,11 +4828,27 @@ im_request(int stream, int version)
 
 					if ((nodeidx > 0) && (nodeidx < pjob->ji_numnodes)) {
 						char *hn;
+						reliable_job_node *rjn = NULL;
 
 						hn  = pjob->ji_hosts[nodeidx].hn_host;
 						snprintf(log_buffer, sizeof(log_buffer),
 						"received IM_ALL_OK prologue hook from host %s", hn?hn:"");
 						log_event(PBSEVENT_DEBUG3, PBS_EVENTCLASS_JOB, LOG_DEBUG, pjob->ji_qs.ji_jobid, log_buffer);
+						/* note that do_tolerate_node_failures() could return 0 if
+                                                 * tolerate_node_failures=job_start but job already moved past
```

Comment:
Is there an extra space before the *?

Reply:
Yes there's one due to this line was filled with spaces while the other lines are tabbed. Fixed.